app/presentation/ImagePinch.js

```diff
@@ -0,0 +1,440 @@
+import React from 'react';
```

Comment:
While this is working nicely, I think using the combination of image + `ScrollView` with `maximumZoomScale` uses an easier and native approach, right?
Can we use this component on Android only?